rumpl/moby
# contrib.mac: Resurrect install bundle script

**- What I did**
Resurrected the script that replaces the binaries on macOS. It now only stores the most recent backup on the user's Desktop. A follow up could improve this.

**- How to verify it**

On macOS: 
1. Build the binaries `DOCKER_LINKMODE=dynamic docker buildx bake`
1. Run the script `./contrib/mac-install-bundle.sh install` 
1. Check the output of `docker version`

```console
 Engine:
  Version:          dev
  API version:      1.43 (minimum version 1.12)
  Go version:       go1.18.4
  Git commit:       HEAD
```

**- Description for the changelog**

N/A
